plugins/slo/pkg/apis/slo/slo.proto

```diff
@@ -196,4 +153,24 @@ message MetricRequest {
   string datasource = 2;
   string serviceId = 3;
   string clusterId = 4;
-}
\ No newline at end of file
+}
+
+message MetricList {
+  repeated Metric items = 1;
+}
+
+message ServiceLevelObjectiveList {
+  repeated SLOImplData items = 1;
+}
+
+enum SLOStatusState {
+  SLO_STATUS_NO_DATA = 0;
```

Comment:
These enum names should be CamelCase, protobuf adds extra prefixes onto the constants it generates so the names won't clash

Reply:
Done